Enterprise-CMCS/eAPD
# Tbolt/4512 Updates to help text in cost allocation for MMIS apds

Resolves #4512 

### Description
Updates help text in cost allocation section for MMIS APDs


### Steps to manually verify this change

1. Create/view MMIS APD
2. Verify Cost Allocate has new text according to design
3. Create/view HITECH APD
4. Verify Cost Allocate help text is unchanged

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
